mne/io/nirx/nirx.py

```diff
@@ -222,7 +253,8 @@ def __init__(self, fname, preload=False, verbose=None):
         def prepend(list, str):
             str += '{0}'
             list = [str.format(i) for i in list]
-            return(list)
+            return (list)
```

Comment:
```suggestion
            return list
```

besides you should not call a variable list